core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1224,6 +1229,29 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleInitPIDRequest(request: RequestChannel.Request): Unit = {
+    // Without AppID, return random 64bit PID.
+
```

Comment:
Doesn't make too much difference, but we could throw `UnsupportedOperationException` for now for all of these methods.